pydantic/typing.py

```diff
@@ -332,7 +332,9 @@ def resolve_annotations(raw_annotations: Dict[str, Type[Any]], module_name: Opti
     annotations = {}
     for name, value in raw_annotations.items():
         if isinstance(value, str):
-            if sys.version_info >= (3, 7):
+            if (3, 10) > sys.version_info >= (3, 9, 8) or sys.version_info >= (3, 10, 1):
```

Comment:
Is this include 3.10 ? isn't it better to do
```suggestion
            if sys.version_info >= (3, 9, 8):
```

Reply:
This issue is not present in version `3.10.0` but presents in the current python `3.10` branch that will become python `3.10.1` on the next release.

For more context, this issue was introduced by this cpython PR - https://github.com/python/cpython/pull/28279

Reply:
Thanks for the info!

Reply:
You're welcome)